src/SFA.DAS.EmployerFinance.Web/Views/Shared/Errors/403.cshtml

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Access denied</title>
+    <meta name="theme-color" content="#0b0c0c" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta property="og:image" content="/Content/lib/govuk-frontend/dist/assets/images/govuk-opengraph-image.png" />
+    <link rel="apple-touch-icon" sizes="180x180" href="/Content/lib/govuk-frontend/dist/assets/images/govuk-apple-touch-icon-180x180.png" />
+    <link rel="apple-touch-icon" sizes="167x167" href="/Content/lib/govuk-frontend/dist/assets/images/govuk-apple-touch-icon-167x167.png" />
+    <link rel="apple-touch-icon" sizes="152x152" href="/Content/lib/govuk-frontend/dist/assets/images/govuk-apple-touch-icon-152x152.png" />
+    <link rel="apple-touch-icon" href="/Content/lib/govuk-frontend/dist/assets/images/govuk-apple-touch-icon.png" />
+    <link rel="mask-icon" href="/Content/lib/govuk-frontend/dist/assets/images/govuk-mask-icon.svg" color="#0b0c0c" />
+    <link rel="shortcut icon" href="/Content/lib/govuk-frontend/dist/assets/images/favicon.ico" type="image/x-icon" />
+    <!--[if !IE 8]><!-->
+    <link href="/Content/lib/govuk-frontend/dist/govuk-frontend-2.0.0.min.css" rel="stylesheet" />
+    <!--<![endif]-->
+    <!--[if IE 8]>
+    <link href="/Content/lib/govuk-frontend/dist/govuk-frontend-ie8-2.0.0.min.css" rel="stylesheet" />
+    <![endif]-->
+    <!--[if lt IE 9]>
+    <script src="/Content/lib/govuk-frontend/dist/html5-shiv/html5shiv.js"></script>
+    <![endif]-->
+    <link href="/Content/css/application.css" rel="stylesheet" />
```

Comment:
The css and images don't exist so the page isn't being styled. Same issue for the other error pages.